normandy/control/static/control/js/components/RecipeForm.jsx

```diff
@@ -3,6 +3,8 @@ import { connect } from 'react-redux'
 import { Link } from 'react-router'
 import { destroy, reduxForm, getValues } from 'redux-form'
 
+import jexl from 'jexl'
```

Comment:
Nit: I think the import structuring we talked about means that this shouldn't have a blank line above it. So `react-router`, `redux-form`, and then `jexl`, with no blanks.